x-pack/typings/numeral.d.ts

```diff
@@ -0,0 +1,20 @@
+/*
```

Comment:
This file was copied from `x-pack/plugins/apm/typings/numeral.d.ts` Adding this file resolves TypeScript check failures that happen with the `x-pack/test/tsconfig.json` config:

```
ERROR x-pack/test failed
      x-pack/plugins/reporting/server/lib/validate/validate_max_content_length.ts:7:21 - error TS7016: Could not find a declaration file for module '@elastic/numeral'. '/dev/shm/workspace/kibana/node_modules/@elastic/numeral/numeral.js' implicitly has an 'any' type.
        Try `npm install @types/elastic__numeral` if it exists or add a new declaration (.d.ts) file containing `declare module '@elastic/numeral';`
      7 import numeral from '@elastic/numeral';
```

I've tried a few other alternative solutions before adding this file:
 - Add `// @ts-ignore` above the import in `validate_max_content_length.ts`
- Import `x-pack/plugins/apm/typings/numeral.d.ts` from `validate_max_content_length.ts`
- Search for any changed imports that would have started causing Reporting plugin code to be scanned in the `x-pack/test/tsconfig.json` config.

I'm open to other suggestions on how to get this PR to pass type check without adding this file.

cc @elastic/kibana-operations @elastic/kibana-platform

Reply:
`@elastic/numeral` v2.5.0 published, with types